lib/domain/usecase/room/create_new_group_chat_interactor.dart

```diff
@@ -31,6 +31,9 @@ class CreateNewGroupChatInteractor {
       );
 
       if (roomId.isNotEmpty) {
+        await matrixClient
+            .getRoomById(roomId)!
+            .setHistoryVisibility(HistoryVisibility.joined);
```

Comment:
IMO, we should change it depend on the kind of the room. For the case of 
- `encrypted`: ok with it?
- `unencrypted`:?
- `public`: ?

Reply:
fixed, set joined for encrypted room only